content/docs/guides/basic-auth.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Basic auth
+---
+
+# Securing Prometheus API and UI endpoints using basic auth
+
+Prometheus does not directly support [basic authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) (aka "basic auth") for connections to Prometheus instances (i.e. to the expression browser or [HTTP API](../../prometheus/latest/querying/api)). If you would like to enforce basic auth for those connections, we recommend using Prometheus in conjunction with a [reverse proxy](https://www.nginx.com/resources/glossary/reverse-proxy-server/) and applying authentication at the proxy layer. You can use any reverse proxy you like with Prometheus, but in this guide we'll provide an [nginx example](#nginx-example).
```

Comment:
i.e. implies a full list, this should be e.g.

Reply:
In this case, though, "to the expression browser or HTTP API" is exhaustive (as far as I know). So the intention is to say "as in" and not "for example."

Reply:
We have many status pages, of which the expression browser is just one. There's also console templates, and various Go debug endpoint.